Pipfile

```diff
@@ -1,92 +1,79 @@
 [[source]]
-url = "https://pypi.python.org/simple"
+url = "https://pypi.org/simple"
 verify_ssl = true
 name = "pypi"
 
 [packages]
 configparser = "*"
 distro = "*"
-pyClamd = "*"
-PyGithub = "*"
-GitPython = "*"
+pyclamd = "*"
+pygithub = "*"
+gitpython = "*"
 phply = "*"
 nltk = "*"
 chardet = "*"
 tblib = "*"
 pdfminer = "*"
-pyOpenSSL = "==22.1.0"
+pyopenssl = "==22.1.0"
 ndg-httpsclient = "*"
 pyasn1 = "*"
 lxml = "*"
-scapy = "==2.5.0rc1"
+scapy = "*"
 cluster = "*"
 msgpack = "*"
 python-ntlm3 = "*"
-Jinja2 = "*"
-Markdown = "*"
+jinja2 = "*"
+markdown = "*"
 psutil = "*"
 ds-store = "*"
 termcolor = "*"
-PyYAML = "*"
+pyyaml = "*"
 tldextract = "*"
-Pebble = "*"
-acora = "*"
+pebble = "*"
 diff-match-patch = "*"
 bravado-core = "*"
 lz4 = "*"
 vulners = "*"
 ipaddresses = "*"
 xdot = "*"
-nocasedict = "*"
+nocasedict = "==1.0.4"
```

Comment:
Could you explain why this specific version is used?

Reply:
Yup. I made a comment in the commit, but there is currently a bug / incompatibility with newer versions: https://github.com/pywbem/nocasedict/issues/139